kotlin/samples/tictactoe/common/src/main/java/com/squareup/sample/gameworkflow/TakeTurnsWorkflow.kt

```diff
@@ -64,32 +86,15 @@ class RealTakeTurnsWorkflow : TakeTurnsWorkflow,
     state: Turn,
     context: RenderContext<Turn, CompletedGame>
   ): GamePlayScreen {
+    val sink = context.makeSink<Action>()
+
     return GamePlayScreen(
         playerInfo = input.playerInfo,
         gameState = state,
-        onEvent = context.onEvent { event ->
-          when (event) {
-            is TakeSquare -> onTakeSquare(state, event)
-            Quit -> emitOutput(CompletedGame(Quitted, state))
-          }
-        }
+        onQuit = { sink.send(Quit) },
```

Comment:
Oh you're doing that here, why the variation?

Reply:
Because I'm not done yet and it's too soon to review.